crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -33,11 +33,11 @@ def initialize(backup)
     def upgrade
       case @version
       when "1.9"
-        knife_files_1_9
+        upgrade_knife_files_1_9
         reset_target_platform_for_nodes
-        crowbar_files_1_9
+        upgrade_crowbar_files_1_9
       when "3.0"
-        network_json_3_0
+        create_network_json_3_0
```

Comment:
Shouldn't this also be "upgrade_"? or at least "recreate"?  Since the JSON file must already be there.